yt/visualization/plot_window.py

```diff
@@ -982,8 +982,8 @@ def _setup_plots(self):
             # colorbar minorticks
             if f not in self._cbar_minorticks:
                 self._cbar_minorticks[f] = True
-            if self._cbar_minorticks[f] is True and MPL_VERSION < LooseVersion('2.0.0') \
-                                                or self._field_transform[f] == symlog_transform:
+            if (self._cbar_minorticks[f] is True and MPL_VERSION < LooseVersion('2.0.0')
+                                                 or self._field_transform[f] == symlog_transform):
```

Comment:
sorry for all the style nits!

can you make sure this new line is aligned properly with the line above?

```
if (self._cbar_minorticks[f] is True and MPL_VERSION < LooseVersion('2.0.0') or
    self._field_transform[f] == symlog_transform):
```